policyengine_canada/variables/gov/cra/tax/income/credits/age_amount/age_amount_credit.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class age_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Age amount credit"
+    unit = CAD
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        p = parameters(period).gov.cra.tax.income.credits.age_amount_credit
+        age_eligibilty = age >= p.age_eligibility
```

Comment:
```suggestion
        eligible = age >= p.age_eligibility
```
Can remove `income_eligibility`